Content/src/Server/ServerGiraffe.fs

```diff
@@ -49,6 +75,8 @@ let webApp =
     |> Remoting.buildHttpHandler
 
 #else
+let getInitCounter () : Task<Counter> = task { return { Value = 42 } }
```

Comment:
let's inline `getInitCounter` both for remoting and the default option - we no longer need it to be a separate function